test/extensions/filters/http/ext_proc/ordering_test.cc

```diff
@@ -134,6 +151,26 @@ class OrderingTest : public testing::Test {
 
   void closeGrpcStream() { stream_callbacks_->onGrpcClose(); }
 
+  // Make it easier to handle buffering
```

Comment:
Bit confused by this comment, is it a TODO or does it just try to justify the helpers below? Maybe just remove?

Reply:
I was trying to comment on the nature of the test helper functions helps in writing tests, but I'm happy to just leave it out.